test/saml20.tests.js

```diff
@@ -23,6 +24,38 @@ describe('saml 2.0 assertion', function () {
     nameIdentifierFormat: 'urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified'
   };
 
+  it('should reject expired cert embedded in saml assertion', function(done){
+    // Turns out the cert we generate expired in 2013
+    let signedAssertion = saml20.create(options);
+    let publicKey = fs.readFileSync(__dirname + '/test-auth0.cer').toString();
```

Comment:
This could probably be moved to a `const` within this describe block since you reuse it.